depth_to_pointcloud/src/depth_to_pointcloud_node.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <image_geometry/pinhole_camera_model.h>
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/image_encodings.hpp>
+#include <sensor_msgs/msg/image.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+#include <sensor_msgs/point_cloud2_iterator.hpp>
+
+#include <limits>
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
> It complains if I put these above the previous block.

Yes .h files should be before the .hpp ones

>  but in the port from ROS1 I left that one as a .h

I see. What is the decision factor here ? If it's a fork we don't rename the files but if it's "porting code" we do? (Not objecting just wondering)

> Unless you have objections, I'll leave this as-is for now.

No I'm fine with it as is that was just surprising when reading the code